src/ts/ConvivaAnalytics.ts

```diff
@@ -45,6 +45,8 @@ export class ConvivaAnalytics {
 
   private readonly logger: Conviva.LoggingInterface = new Html5Logging();
 
+  private duration: number = 0;
```

Comment:
Maybe a comment would be good why this is needed. Could also be good to rename the variable to something like `mainContentDuration` or `sourceDuration`, given that we have main content duration, ad duration, and ad break duration.

Reply:
Renamed https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/118/commits/a0f340727fe1816302b51f74011dd2ef6d801ab6.